### PR TITLE
Basic Tutorial fixes

### PR DESCRIPTION
**Pull request by hessenfarmer**
Started to streamline the basic tutorial re #4365

### CODE REVIEW

**Comment by hessenfarmer** [comment]
@Noordfrees 
Any chance this could make it to v1.0?
it improves the tutorial like suggested. Although not that radical but I believe it is definitly an improvement

File: data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua [hunk 1]
```diff
@@ -24,35 +22,24 @@ scold_player = {
...
 initial_message_01 = {
    title = _"Welcome to the Widelands Tutorial!",
    body = (
       h1(_"Welcome to Widelands!") ..
       li_image("images/logos/wl-ico-64.png",
-         _[[Widelands is a slow-paced build-up strategy game with an emphasis on construction rather than destruction. This tutorial will guide you through the basics of the game.]]) ..
-      li_arrow(_[[You will be guided through this tutorial by objectives]]) ..
-      li(_[[Follow the intructions in the objective below so that I can show you where to find them.]])
+         _[[Widelands is a slow-paced build-up strategy game with an emphasis on construction rather than destruction.]]) ..
+      li_arrow(_[[You will be guided through this tutorial by objectives]])
```
Comment (Noordfrees) [inline data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua]:
```suggestion
      li_arrow(_[[You will be guided through this tutorial by objectives.]])
```

File: data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua [hunk 2]
```diff
@@ -24,35 +22,24 @@ scold_player = {
...
    number = 1,
-   body = objective_text(_"Closing this window",
-      p(_[[This is the ‘Objectives’ window. You can return to this window for instructions at any time.]]) ..
-      li_image("images/wui/menus/objectives.png", _[[ You can open and close this window by clicking on the ‘Objectives’ button in the toolbar on the bottom of the screen.]]) ..
-      li_arrow(_[[Like any other window, you can also close the ‘Objectives’ window by right-clicking on it.]]) ..
-      li_arrow(_[[When you have accomplished an objective, it will disappear from the list above.]]) ..
-      li(_[[Try it out.]])
+   body = objective_text(_"Handling of the ‘Objectives’ window",
+      li_image("images/wui/menus/objectives.png", _[[ You can open and close the ‘Objectives’ window by clicking on the ‘Objectives’ button in the toolbar on the bottom of the screen.]]) ..
+      li_arrow(_[[Like any other window, you can also close the window by right-clicking on it.]]) ..
```
Comment (Noordfrees) [inline data/campaigns/tutorial01_basic_control.wmf/scripting/texts.lua]:
```suggestion
      li_arrow(_[[Like most other windows, you can also close the window by right-clicking on it.]]) ..
```

**Review by Noordfrees — COMMENTED** [review]
LGTM, two comments inline.

---

I'm not sure if in the expression
```
_"foo %s":bformat("bar")
```
will be interpreted like
```
_("foo %s"):bformat("bar")
```
or
```
_("foo %s":bformat("bar"))
```
It's safer to use extra `()` to disambiguate.

**Comment by hessenfarmer** [comment]
> 
> 
> LGTM, two comments inline.
> 
> I'm not sure if in the expression
> 
> ```
> _"foo %s":bformat("bar")
> ```
> 
> will be interpreted like
> 
> ```
> _("foo %s"):bformat("bar")
> ```
> 
> or
> 
> ```
> _("foo %s":bformat("bar"))
> ```
> 
> It's safer to use extra `()` to disambiguate.

committed the suggestions. regarding the bformat we should merge your branch to do this first and then resolve the merge conflicts in here.

**Comment by hessenfarmer** [comment]
ok resolved the merge conflicts. Should be good to go now

**Review by Noordfrees — APPROVED** [review]

**Pull request merged**